doc/3/controllers/document/m-create/index.md

```diff
@@ -0,0 +1,70 @@
+---
+code: true
+type: page
+title: mCreate
+description: Creates multiple documents
+---
+
+# mCreate
+
+Creates multiple documents in the provided index and collection.
+
+---
+
+## Arguments
+
+```java
+public CompletableFuture<ConcurrentHashMap<String, Object>> mCreate(
+      final String index,
+      final String collection,
+      final ArrayList<ConcurrentHashMap<String, Object>> documents)
+throws NotConnectedException, InternalException
+
+public CompletableFuture<ConcurrentHashMap<String, Object>> mCreate(
+      final String index,
+      final String collection,
+      final ArrayList<ConcurrentHashMap<String, Object>> documents,
+      final Boolean waitForRefresh)
+throws NotConnectedException, InternalException
+```
+
+| Arguments          | Type                                                    | Description                       |
+| ------------------ | ------------------------------------------------------- | --------------------------------- |
+| `index`            | <pre>String</pre>                                       | Index                             |
+| `collection`       | <pre>String</pre>                                       | Collection                        |
+| `documents`        | <pre>ArrayList<ConcurrentHashMap<String, Object>></pre> | ArrayList containing the documents to create |
+| `waitForRefresh`   | <pre>Boolean</pre> | Optional parameters                |
+
+---
+
+### documents
+
+Each document has the following properties:
+
+| Arguments          | Type                                         | Description                       |
+| ------------------ | -------------------------------------------- | --------------------------------- |
+| `_id`              | <pre>String</pre> (optional)                 | Optional document ID. Will be auto-generated if not defined.             |
+| `body`             | <pre>Boolean</pre> (optional)                | Document body |
+
+## Return
+
+A `ConcurrentHashMap` which has a `successes` and `errors` array:
```

Comment:
You must provide the exact type for the successes and errors values, otherwise they aren't usable